code/lib/core-server/src/dev-server.ts

```diff
@@ -29,7 +29,7 @@ export async function storybookDevServer(options: Options) {
     options.presets.apply<CoreConfig>('core'),
   ]);
 
-  const serverChannel = getServerChannel(server);
+  const serverChannel = await options.presets.apply('serverChannel', getServerChannel(server));
```

Comment:
Should we call this `experiemental_xyz` for now?

Also, the name `serverChannel` is a little bit unintuitive to me. That implies I'd be providing rather than modifying a server channel. I'm not sure of a better name though.

Reply:
Any preset value gets the default value, and gets the option to do something with it, modify it, return something new.

I don't have a better name either..